### PR TITLE
[Doc] Add doc of save & load low bit model in CPU

### PR DESCRIPTION
**Pull request by zhentaoyu**
## Type of Change

feature or bug fix or documentation or others
API changed or not: no

## Description

detail description 
JIRA ticket: xxx
add doc  for #1055

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

**Review by kevinintel — COMMENTED** [review]
add real hf model path

File: docs/weightonlyquant.md [hunk 1]
```diff
@@ -115,3 +115,24 @@ gen_ids = woq_model.generate(input_ids, max_new_tokens=32, **generate_kwargs)
...
+model_path = "meta-llama/Llama-2-7b-chat-hf" # your_pytorch_model_path_or_HF_model_name
+saved_dir = "4_bit_llama2" # your_saved_model_dir
+# quant
+model = AutoModelForCausalLM.from_pretrained(model_path, load_in_4bit=True, use_llm_runtime=False)
+# save quant model
+model.save_pretrained(saved_dir)
+# load quant model
+loaded_model = AutoModelForCausalLM.from_pretrained(saved_dir)
+```
+| Inference Framework |   load GPT-Q  |    load fp32 and quantized model    |
```
Comment (kevinintel) [inline docs/weightonlyquant.md:133]: load gptq model    load the saved low-precision model from save_pretrained

**Review by kevinintel — APPROVED** [review]

**Review by kevinintel — APPROVED** [review]

File: docs/weightonlyquant.md (hunk 1 above)
Comment (zhentaoyu) [reply docs/weightonlyquant.md]: refined it in [f3b58cb](https://github.com/intel/intel-extension-for-transformers/pull/1071/commits/f3b58cb814c891a2b4f33801d303f936afa9ed0d)

File: docs/weightonlyquant.md [hunk 2]
```diff
@@ -115,3 +115,24 @@ gen_ids = woq_model.generate(input_ids, max_new_tokens=32, **generate_kwargs)
...
+# quant
+model = AutoModelForCausalLM.from_pretrained(model_path, load_in_4bit=True, use_llm_runtime=False)
+# save quant model
+model.save_pretrained(saved_dir)
+# load quant model
+loaded_model = AutoModelForCausalLM.from_pretrained(saved_dir)
+```
+| Inference Framework |   Load GPT-Q model from HuggingFace |  Load the saved low-precision model from ITREX |
+|:--------------:|:----------:|:----------:|
+|       llm_runtime (use_llm_runtime=True)      |  &#10004;  |  &#10004;  |
```
Comment (hshen14) [inline docs/weightonlyquant.md:135]: llm_runtime -> LLM Runtime (use_llm_runtime=True)?

Comment (zhentaoyu) [reply docs/weightonlyquant.md]: thanks! fixed in https://github.com/intel/intel-extension-for-transformers/pull/1071/commits/62f33283cf8f25af184786b7aa57e6877ccc9c78

**Pull request merged**